selectors/general.js

```diff
@@ -13,3 +13,9 @@ export function areTimezonesEnabledAndSupported(state) {
     const config = getConfig(state);
     return config.ExperimentalTimezone === 'true';
 }
+
+export function getBasePath(state) {
+    const config = getConfig(state) || {};
+
+    return new URL(config.SiteURL || window.location.href).pathname;
```

Comment:
I'm not sure if this makes sense to use window.location.href here. It looks like that might send you to `/<team>/channels/<channel>` if you're viewing a channel when this is called without. Should we instead use window.location.origin if the SiteURL isn't configured without worrying about extracting the pathname?

Reply:
You're 100% right. Fixed.